### PR TITLE
Removed resource public IP code and all references to the resource fr…

### PR DESCRIPTION
**Pull request by saloneerege**
…om .tf file and documentation

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,5 @@ terraform.tfstate
 *.backup
 *.bak
 *~
+*.tf
```
Comment (rchadalawada) [inline .gitignore]: why? don't we want to commit .tf files in the example folder?

Comment (shylasrinivas) [reply .gitignore]: Agree. Main.tf should not be in .gitignore.

**Review by shylasrinivas — COMMENTED** [review]
can you remove /*.tf from gitignore? I checked other providers and there is no reference to *.tf in other gitignore files.

**Comment by saloneerege** [comment]
> can you remove /*.tf from gitignore? I checked other providers and there is no reference to *.tf in other gitignore files.

I wanted that added since we dont want developers checking in their local .tf files. I think it is okay for other providers not to have it in their .gitignore but I think we need it.

**Review by rchadalawada — APPROVED** [review]

**Pull request merged**